scripts/assign.py

```
@@ -207,70 +207,146 @@ def main():
-                    requests.post(f"{issue_url}/comments", headers=headers, json={"body": comment_body})
-                    return
-
-                # Assign the issue
-                assignees_url = f"https://api.github.com/repos/{owner}/{repo}/issues/{issue_number}/assignees"
-                print(f"Assigning issue via {assignees_url}")
-                assign_response = requests.post(assignees_url, headers=headers, json={"assignees": [user_login]})
-                if assign_response.status_code >= 400:
-                    print(f"Error assigning issue: {assign_response.status_code} - {assign_response.text}")
-                    return
+                    print(f"Denying assignment due to issue #{issue_without_pr} without PR")
+                    requests.post(f"{issue_url}/comments", headers=headers, json={"body": rejection_msg})
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)